core/src/main/java/org/frankframework/management/bus/endpoints/Monitoring.java

```diff
@@ -131,7 +129,9 @@ public Message<String> addMonitorOrTrigger(Message<?> message) {
 		try {
 			monitor.configure();
 		} catch (ConfigurationException e) {
-			throw new BusException("unable to (re)configure Monitor", e);
+			log.info("Unable to (re)configure monitor [{}]", monitor.getName(), e);
```

Comment:
NB: Despite the exception we catch, the invalid trigger has already been added and we are stuck with that invalid trigger until we update it.

Ideally we should validate the TriggerDTO before creating and adding the trigger, also on update.

Not sure if we should do that as part of this PR, or a follow-up PR.

Reply:
Is it not possible to remove it if we get an exception?

Reply:
Zoals de code nu in elkaar zit is dat niet makkelijk.

Reply:
I now made a change to remove the trigger but it would feel better to me if we would actually have a validation-step before adding it.
I'll do that in a separate PR so you can see if you're OK with the changes I have in mind for that.